test/basic/test.ts

```diff
@@ -51,25 +52,6 @@ test('should detect title, description and keywords even when they are in the bo
   expect(result).toEqual(expected)
 })
 
-test('should detect title, description and keywords', async () => {
```

Comment:
Can we keep this test please

Reply:
It's not quite a duplicate test because we have separate tests for tags in/out the body tag.

Reply:
I understand. put it back together.
fixed: b4e451a